### ISSUE TITLE
onScroll gets called unnecessarily, making scrollbar flash

### ISSUE THREAD

**Issue by FSDRE** [KingSora/OverlayScrollbars#191]
Hi,

first of all I made a repo for you in oder to reproduce the issue with a makeshift tab control using Angular and the overlay scrollbars ngx wrapper:

https://github.com/FSDRE/overlaybars

**The issue can be reproduced with:**

* Firefox
* Edge
* Chrome on Android

Chrome on desktop seems to work but I am pretty certain I saw this behavior there also in other circumstances.

**Used Overlay Scrollbar Options:**

```typescript
public scrollerOptions: any = {
    className: 'os-theme-thin-dark',
    paddingAbsolute: true,
    overflowBehavior: {
      x: 'scroll',
      y: 'hidden'
    },
    scrollbars: {
      autoHide: 'scroll',
      autoHideDelay: 500
    },
    callbacks: {
      onScroll: this.refreshScroller.bind(this),
      onOverflowChanged: this.refreshScroller.bind(this),
      onOverflowAmountChanged: this.refreshScroller.bind(this)
    }
  };
```

**What's right? What's wrong?**

If you fire up the app with `npm run serve` you see a little scrollable tab control. You can scroll with the little arrows or by grabbing the scrollbar when active. I defined `autoHide: 'scroll',` so the scrollbar should only be visible whenever scrolling is active.

When scrolled completely to the left (`viewport.scrollLeft == 0`) you can toggle between tab 1 and tab 2. Nothing happens, no scrollbar is shown, which is absolutely correct since the overflow or the scroll position do not change. A selected tab is equally wide compared to an unselected one.

Now use the little arrow to scroll to the right so that you can see tab 4 and 5 for example. Now toggle between those two tabs. The scrollbar gets shown everytime you toggle between the tabs. This should not happen.

onScroll gets called 3 times on a tab change. If I console.log the state of the scrollbar instance, there are absolutely no changes to any property whatsoever, so I wonder why onScroll gets called in the first place.

Hint: the `refreshScroller()` method only makes the little arrows appear or disappear, it does not change the Overlay Scrollbar instance or anything.

If you need anything else, please let me know! Thanks for your time and the awesome control!

**Comment by KingSora** [comment]
Good day!

Thanks for your detailed Repo, after a pretty long investigation I found the cause of this behavior. I won't explain it because the text would be quite long and you would need more knowledge about the project to fully understand it.

I can give you a (temporary) workaround until I've figured out how to fix it. If you set a fixed height on the tabs (`height: 60px`) for example, everything should work as expected.

**Comment by FSDRE** [comment]
Big thanks for the investigation! I am going to test the workaround later today.

**Comment by FSDRE** [comment]
The workaround does not seem to work on my side. Tested on Firefox and Edge.

I tried `height: 65px` on the div with `class="tabs"` and also on the individual divs with `class="tab"`. Still got the flashing going on. It is not a serious issue in our case, just a little optical bug. And if there is a bugfix in the future it's all good.

**Comment by KingSora** [comment]
@FSDRE The fixed height has to be applied to the `#scroller` element.

**Comment by FSDRE** [comment]
@KingSora Ahhhhhhhhhh, we have liftoff! It works! Thanks a bunch!

**Comment by KingSora** [comment]
@FSDRE It should be fixed in [version 1.11.0](https://github.com/KingSora/OverlayScrollbars/releases/tag/v1.11.0). :)

**Comment by FSDRE** [comment]
Perfect, keep up the good work!
